src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java

```diff
@@ -32,12 +32,11 @@
 
 import static com.github.kagkarlsson.scheduler.ExecutorUtils.defaultThreadFactoryWithPrefix;
 
-public class Scheduler implements SchedulerClient {
+public class Scheduler extends SchedulerClient.StandardSchedulerClient {
 
```

Comment:
I see. Which one would you prefer?
* delegating to internal `StandardSchedulerClient`
* make an `AbstractSchedulerClient` and extend from that in both `StandardSchedulerClient` and `Scheduler`